assets/src/edit-story/components/library/panes/media/mediaPane.js

```diff
@@ -252,6 +253,24 @@ function MediaPane(props) {
     [hasMore, isMediaLoading, isMediaLoaded, setNextPage]
   );
 
+  // The media pane is supposed to have a scrollbar that shows on focus and
```

Comment:
Maybe separate this comment into:

// TODO(#issue): Implement scrollbar that only appears on MediaPane focus.
// Recalculates padding of Media Pane so it stays centered ... As of 
// May 2020 this cannot be achieved without js (as the scrollbar-gutter prop is no yet ready).